test/stepDefinitions/emptySteps.js

```diff
@@ -0,0 +1,15 @@
+
+var { defineSupportCode } = require("cucumber");
+
+/**
+ * This class is to put steps that require no implementation but are there purely for readability on the BDD side
+ */
+defineSupportCode(function ({ Given, When, Then}) {
+
+  Given(/^this event contains a read only value of the fixed list from the previous event$/, function() {
+    //The (Approve case) event has been configured in the definition file to have a read only field containing the value
```

Comment:
is this a good practice?

Reply:
I have taken this out for now however if you generalise a scenario to much it can be good for reusability but can sometimes loose the more niche edge case details of whats going on, this detail was originally abstracted out of another step to make it more reusable but this line is the defining point of the test, or the what caused the bug in the first place depending on how you want to look at it. 

I would say it's not bad practise to do this if it is bringing value to the reader but there may not necessarily be anything to implement, since we should typically be coming up with bdds in 3As there may be situations where there is no relevant code implementation to a step (perhaps it's been done a definition config already for example) but having the blank step there makes the scenario read better